build/utils/npm.go

```diff
@@ -192,7 +210,8 @@ func GetNpmVersion(executablePath string, log utils.Log) (*version.Version, erro
 }
 
 type NpmTreeDepListParam struct {
-	Args []string
+	Args               []string
+	InstallCommandArgs []string
```

Comment:
Let's add a description for each of the above propeties, to help the reader understand the difference between the two.